test_scripts/API/AutoCompleteList/AutoCompleteList.txt

```diff
@@ -0,0 +1,34 @@
+./test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua 

```

Comment:
@Aleksandriys , next cases also should be covered and added to test set list:
- **autoCompleteList  wrong type** (currently covered only array value wrong type)
- **autoCompleteList array out upper bound** (this is in case 019_ATF_SetGlobalProperties_N_AutoCompleteList_ArrayValuesOutOfUpperBound.lua is related to array value, but if mentioned script is covering "**array** out upper bound", then "array **value** out upper bound" case is not covered)